scripts/repo/charts/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Isn't this file getting overriden in `create-repo.sh`?
`curl -s https://nuodb.github.io/nuodb-helm-charts/index.yaml > repo/charts/index.yaml`

So, it should be in `.gitignore`.

Reply:
I'm going to adjust the readiness probe so that this file is not required.